meson_options.txt

```
@@ -67,3 +67,10 @@ option(
     value: 'auto',
     description: 'Set rpath for executables (automatically enabled if the prefix is not /usr)',
 )
+
+option(
+    'build_macos_app',
+    type: 'feature',
+    value: 'disabled',
```

This option isn't auto detected, so that a boolean is more appropriate:
```suggestion
    type: 'boolean',
    value: false,
```